.github/workflows/pull_request.yml

```diff
@@ -199,3 +199,92 @@ jobs:
         with:
           name: ${{ matrix.os }}-${{ matrix.compiler }}-nebula-test-logs
           path:  ./build/server_*/logs/
+
+  standalone:
```

Comment:
I think you could build standalone nebula in origin ci job and test after origin test step.

Reply:
Another compile workflow is needed.And there is a metirc to combine workflows in origin ci job. I don't want this too heavy.